instrumentation/aws-sdk/aws-sdk-1.11/library/build.gradle.kts

```diff
@@ -34,3 +34,13 @@ if (!(findProperty("testLatestDeps") as Boolean)) {
     }
   }
 }
+
+tasks {
+  val testStableSemconv by registering(Test::class) {
+    jvmArgs("-Dotel.semconv-stability.opt-in=database")
+  }
+
+  check {
+    dependsOn(testStableSemconv)
+  }
+}
```

Comment:
can you add to the javaagent module also?

Reply:
@trask done